evap/evaluation/tests/test_models.py

```diff
@@ -349,11 +350,14 @@ def test_can_be_deleted_by_manager(self):
         mommy.make(Contribution, contributor=contributor)
         self.assertFalse(contributor.can_be_deleted_by_manager)
 
+        proxy_user = mommy.make(UserProfile, is_proxy_user=True)
+        self.assertFalse(proxy_user.can_be_deleted_by_manager)
+
     def test_inactive_users_hidden(self):
         active_user = mommy.make(UserProfile)
         mommy.make(UserProfile, is_active=False)
 
-        self.assertEqual(list(UserProfile.objects.exclude_inactive_users().all()), [active_user])
+        self.assertEqual(list(UserProfile.objects.exclude(is_active=False).all()), [active_user])
```

Comment:
When you are already changing this line, could you also remove the .all() call? It should not be required.